test/integration/evictions/evictions_test.go

```diff
@@ -334,6 +340,85 @@ func TestEvictionVersions(t *testing.T) {
 	}
 }
 
+// TestEvictionWithFinalizers tests eviction with the use of finalizers
+func TestEvictionWithFinalizers(t *testing.T) {
+	cases := map[string]struct {
+		enablePodDisruptionConditions bool
+		phase                         v1.PodPhase
+	}{
+		"terminal pod with job backoff policy enabled": {
+			enablePodDisruptionConditions: true,
+			phase:                         v1.PodSucceeded,
+		},
+		"terminal pod with job backoff policy disabled": {
+			enablePodDisruptionConditions: false,
+			phase:                         v1.PodSucceeded,
+		},
+		"running pod with job backoff policy enabled": {
+			enablePodDisruptionConditions: true,
+			phase:                         v1.PodRunning,
+		},
+		"running pod with job backoff policy disabled": {
```

Comment:
nit: this gate is probably not `job backoff` but related to adding additional conditions about reason of pod eviction. In that context, this description is confusing

Reply:
thanks, updated.